### DISCUSSION TITLE
How to install on pcs without yarn or anything

### DISCUSSION THREAD

**Discussion by TTR-maniac** [matteobruni/tsparticles#1394] [General]
https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTFh_OYPH1K3Eu3yhAIOC8Au_VLSGTSYrrtNgPwLUIrUwBW9MFcssrq9pU5Iu20pl6p9XeYrXzqkGyUSg8L:https://ghostslol.files.wordpress.com/2020/04/person-s-gray-hoodie-1482476.jpg%3Fw%3D4000&usqp=CAU
-_-
HOW PLS HELPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPP

**Comment by matteobruni** [comment]
Hi @TTR-maniac,

What do you mean with installing on pcs? I don’t understand what you are asking.

**Reply by mgreci** [reply]
Do you mean using a CDN instead of a package manager?

**Comment by T1mm1Turner** [comment]
You can curl the raw "tsparticles.bundle.min.js" from the unpkg page (curl https://unpkg.com/browse/tsparticles/ > tsparticles.bundle.min.js) in a bash or any shell.
